vyper/ast/parse.py

```diff
@@ -470,13 +474,11 @@ def visit_UnaryOp(self, node):
         self.generic_visit(node)
 
         is_sub = isinstance(node.op, python_ast.USub)
-        is_num = (
```

Comment:
i think this is actually just dead code now right?

Reply:
hmm, the coverage reports indicates the branch is getting hit https://app.codecov.io/gh/vyperlang/vyper/blob/master/vyper%2Fast%2Fparse.py#L478